draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -11868,6 +11868,15 @@ Content-Encoding: gzip
    <xref target="RFC4918" x:fmt="of" x:sec="11.2"/>) because of its general
    applicability (<xref target="status.422"/>).
 </t>
+<t>
+   The grammar definition for the Via field's "received-by" was unfortunately
```

Comment:
"unfortunately" reads a little weird here. Not objecting strongly, but this para could be massaged a bit.